docs/migrating_to_6.md

```diff
@@ -130,7 +130,7 @@ await q;
 await q.clone(); // Can `clone()` the query to allow executing the query again
 ```
 
-<h2 id="model-exists-returns-a-lean-document-instead-of-boolean"><a href="#model-exists-returns-a-lean-document-instead-of-boolean">Model.exists(...) now returns a lean document instead of boolean</a></h2>
+## Model.exists(...) now returns a lean document instead of boolean {#model-exists-returns-a-lean-document-instead-of-boolean}
```

Comment:
```suggestion
## `Model.exists(...)` now returns a lean document instead of boolean {#model-exists-returns-a-lean-document-instead-of-boolean}
```

not in original but would look better